enterprise/models.py

```diff
@@ -1823,6 +1823,13 @@ def encrypted_password(self):
             )
         return self.decrypted_password
 
+    @encrypted_password.setter
+    def encrypted_password(self, value):
```

Comment:
As we discussed offline, the confusion around the encrypted vs decrypted is a bit confusing. Please put some comments around either the fields or the properties related to those fields describing how the data is only encrypted at rest (i.e. in the DB) and is *always* unencrypted in code (unless we manually encrypt it).